include/natalie/string_object.hpp

```diff
@@ -223,7 +224,9 @@ class StringObject : public Object {
     Value reverse_in_place(Env *);
     Value rstrip(Env *) const;
     Value rstrip_in_place(Env *);
+    size_t char_count(Env *) const;
```

Comment:
Any chance this is cached somewhere? There are a lot of string methods that play around with indices, and you'll need to know how to handle negative indices by wrapping the character count. I wasn't sure if I just missed it somewhere.

Reply:
No, not cached, but I will take note to do that later!